stl/inc/xcharconv_ryu.h

```diff
@@ -400,21 +406,21 @@ inline void __append_n_digits(const uint32_t __olength, uint32_t __digits, char*
     __digits /= 10000;
     const uint32_t __c0 = (__c % 100) << 1;
     const uint32_t __c1 = (__c / 100) << 1;
-    _CSTD memcpy(__result + __olength - __i - 2, __DIGIT_TABLE + __c0, 2);

```

Comment:
Why is `_CSTD` qualification being removed in this file? We conventionally qualify all calls to non-`_Ugly` functions. (I also observe that it's not being removed consistently.)